Console/Command/QueueShell.php

```diff
@@ -11,7 +11,9 @@
 
 App::uses('Folder', 'Utility');
 App::uses('File', 'Utility');
-App::import('Vendor', 'Hipchat', array('file' => 'HipchatPhp/src/HipChat/HipChat.php')); 
+if (!class_exists('HipChat\HipChat')) {
```

Comment:
Hm... hold on - I'm feeling suspicious too.
Hipchat isn't a vanilla PHP file with a class in it... it's a PSR compliant composer project. Isn't there an autoloader that does this for us?